examples/default-values-backend/README.md

```diff
@@ -0,0 +1,81 @@
+# Contentful Backend Demo APP
+
+A demo App that will prefill certain values when an entry is created.
+
+The purpose of this demo is to show how backend Apps can be setup and used.
+
+## Prerequisites
+
+This demo requires an up to date version of node.js and yarn.
+
+In addition you will need a way to make the running app accessible over http, so
+that it can receive events from contentful. If you want to run the App locally,
+we suggest using [ngrok](https://ngrok.com/) for this. It is a free tool that
+allows you to easily create a url that allows access to applications served from localhost.
+
+## Setup instructions
+
+1. Run
+
+```shell
+ngrok http 3543
+```
+
+Take note of the url provided by NGROK. This step is only necessary when running
+the app locally.
+
+Although we are running the `http` command - ngrok will provide a https address.
+We will need to use this secure URL for our app setup to work correctly.
+
+2. Update `.env` with
+
+```shell
+CMA_TOKEN= # You can obtain this token from the contentful web app
```

Comment:
Do we need to provide information about how to get this?

Reply:
I left it unchanged from what @Jwhiles added. When I tried finding the web app page myself I struggled a bit, however, so additional information would be nice.
I can't directly link the URL (/spaces/<space>/api/cma_tokens) because the space ID isn't clear.
Maybe a simple description of which buttons in the UI to click to get to the CMA token page would work?